### ISSUE TITLE
[FEATURE] Accessibility support for screen readers

### ISSUE THREAD

**Issue by denissariyannis** [sdatkinson/NeuralAmpModelerPlugin#607]
On macOS the Snapshot plugin is currently not accessible for screen reader users.
macOS owns a build-in screen reader called VoiceOver which lets blind users have access to the Mac. I loaded the plugin into Logic Pro but the UI of the plugin is not accessible.
Logic Pro provides the opportunity to switch the view of the plugin to Controls view. There, the user has direct access to the parameters and its values. This view is accessible with VoiceOver. Right now, the parameters which the plugin exposes are very limited. Input, Threshold, Bass, Middle, Treble, etc.
I would have access to the plugin if other functions also would be showing up, such as buttons to load models and IRs.

Currently there is no alternative for me to use NAM-models. Hardware devices which can play them usually come with touch screens / non speaking menus or with inaccessible apps.

**Comment by sdatkinson** [comment]
Hi @denissariyannis .

Thanks for sharing with me how this is usually handled.

Are there any other "file-loading" plugins (e.g. IR loaders) that work? Interested to see how they do it.

The problem, I suspect, is that the filename isn't a "parameter"--to my knowledge (and I haven't actually really studied plugin standards closely), plugin parameters are only numeric-type (floating point, ultimately), which works for things like switches, options from a pre-set list, and continuous things like gain.

But that's just me thinking about the solution you started with. Perhaps there are other ways this is tackled? I admittedly have little experience with UIs!

**Comment by jquesada2016** [comment]
Hello! Yes, another visually impaired user here. Screen reader support would be fantastic.
